docs/community/release-checklist.md

```diff
@@ -131,11 +131,6 @@ The [gofish][] package manager has automation in place to create an update when
 
 Adding new versions to [choco][] is automated, but you can check the status of package approval and publishing at the [aks-engine chocolatey page][choco-status].
 
-Note that it can take a while for the NuGet machinery to submit the current release, and it may be reviewed for several days before it is actually made available.
```

Comment:
Just to explain for posterity: now that we're a blessed package, the choco review process is not time-consuming and it never waits in review.